hi-ml/testhiml/testhiml/test_run_ml.py

```diff
@@ -374,19 +374,6 @@ def test_model_weights_when_resume_training() -> None:
             assert recovery_checkpoint == runner.checkpoint_handler.trained_weights_path
 
 
-def test_runner_end_to_end() -> None:
```

Comment:
Why was this test removed?

Reply:
it's not testing anything; I accidently merged it in a previous PR

Reply:
This is the end to end test https://github.com/microsoft/hi-ml/blob/c4e25b8a797cfee3822bda1e1c05e399667bf68e/hi-ml/testhiml/testhiml/test_run_ml.py#L292 
I started writing an end to end test and then found out it was already there but forgot to delete my code